index.js

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+const colors = require('colors');
+const fs = require('fs');
+const program = require('commander');
+const exec = require('shelljs').exec;
+const path = require('path');
+
+/**
+* Init program version
+*/
+const packageJSON = JSON.parse(fs.readFileSync('package.json'));
```

Comment:
Good idea, but seems that this approach will work only when you start with npm, but will not work any longer for global call.